site/_docs/howto.md

```diff
@@ -99,6 +99,9 @@ $ ./gradlew spotlessCheck checkstyleAll # report code style violations
 
 You can use `./gradlew assemble` to build the artifacts and skip all tests and verifications.
 
+You might find these two links about Gradle are useful: [Gradle five things](https://docs.gradle.org/current/userguide/what_is_gradle.html#five_things)
```

Comment:
From a quick look the links contain general information about Gradle. They are not Calcite specific and from what I see not specific to tests either. Moving them to _Gradle vs Gradle wrapper_ seems more appropriate. 

Also I would reformulate as follows:
For more information about Gradle check the following links:

Reply:
Thanks for suggestion. I reformulated as suggested.

Regarding to moving it to `Gradle vs Gradle wrapper`: my intention to add this new pieces because I tried to run `./gradlew check` locally but it failed to run some modules's tests locally. So I spent time to research how to run `:core`'s check only, which I thought might be useful tip for more people who cannot run a full check locally. So seems to me that these new pieces are more relevant here (which shares some common commands to developers).

@zabetak if you think it still better to be moved to `Gradle vs Gradle wrapper`, I am change it.

Reply:
Remove the "[Document]" prefix. That's not a prefix we use.